voluptuous/tests/tests.py

```diff
@@ -8,6 +8,8 @@
     validate, ExactSequence, Equal, Unordered
 )
 from voluptuous.humanize import humanize_error
+from voluptuous.encoding import to_utf8_py2
+import six
```

Comment:
Can we get this done without importing `six`? Because Ideally we don't want voluptuous to have external dependencies. Check [this](https://docs.python.org/2/howto/unicode.html#the-unicode-type)